docs/usage.md

```diff
@@ -475,6 +480,101 @@ We recommend adding the following line to your environment to limit this (typica
 NXF_OPTS='-Xms1g -Xmx4g'
 ```
 
+## Tutorials
+
+### Tutorial - How to create your custom database
+
+#### Kraken2
+
+Kraken2 allows the user to build custom databases.
+
+To install a taxonomy:
+
+```bash
+kraken2-build --download-taxonomy --db $DBNAME
+```
+
+To install one or more reference libraries:
+
+```bash
+--download-library bacteria --db $DBNAME
+--download-library viral --db $DBNAME
+--download-library archaea --db $DBNAME
+```
+
+To add more genomes:
+
+```bash
+kraken2-build --add-to-library genome.fa --db $DBNAME
+```
+
+You can follow Kraken2 [tutorial](https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown#custom-databases) for a more detailed description.
+
+#### Centrifuge
+
+Centrifuge allows the user to [build custom databases](https://ccb.jhu.edu/software/centrifuge/manual.shtml#custom-database). The user should download taxonomy files, make custom `seqid2taxid.map` and combine the fasta files together.
+
+```bash
+centrifuge-download -o taxonomy taxonomy
+
+## custom seqid2taxid.map
+NC_001133.9    4392
+NC_012920.1    9606
+NC_001134.8    4392
+NC_001135.5    4392
+
+cat *.{fa,fna} > input-sequences.fna
+centrifuge-build -p 4 --conversion-table seqid2taxid.map --taxonomy-tree taxonomy/nodes.dmp --name-table taxonomy/names.dmp input-sequences.fna taxprofiler_cf
+```
+
+#### Kaiju
+
+It is possible to [create custom databases](https://github.com/bioinformatics-centre/kaiju#custom-database) with Kaiju.
+
+```bash
+kaiju-mkbwt -n 5 -a ACDEFGHIKLMNPQRSTVWY -o proteins proteins.faa
+kaiju-mkfmi proteins
+```
+
+#### MALT
+
+To create a custom database for MALT, the user should download and unzip the following database which lists all NCBI records. The input files are specified using -i and the index is specified using -d. A detailed description for each argument can be found [here](https://software-ab.informatik.uni-tuebingen.de/download/malt/manual.pdf)
+
+```bash
+wget https://software-ab.informatik.uni-tuebingen.de/download/megan6/megan-nucl-Feb2022.db.zip
+unzip megan-nucl-Feb2022.db
+malt-build -i path/to/fasta/files/*.{fna,fa} -s DNA -d index -t 8 -st 4 -a2t megan-nucl-Feb2022.db
```

Comment:
```suggestion

MALT does not provide any default databases for profiling, therefore you must build your own.

You need FASTA files to include, and an (unzipped) [MEGAN mapping 'db' file](https://software-ab.informatik.uni-tuebingen.de/download/megan6/) for your FASTA type.

In addition to the input directory, output directory, and the mapping file database, you also need to specify the sequence type (DNA or Protein) with the `-s` flag.

```bash
malt-build -i <path>/<to>/<fasta>/*.{fna,fa,fasta} -a2t <path>/<to>/<map>.db -d <YOUR_DB_NAME>/  -s DNA
```

You can then add the `<YOUR_DB_NAME>/` path to your nf-core/taxprofiler database input sheet.

> ⚠️ MALT generates very large database files and requires large amounts of RAM. You can reduce both by increasing the step size `-st` (with a reduction in sensitivity).

> 🛈 MALT-build can be multi-threaded with `-t` to speed up building.

See the [MALT manual](https://software-ab.cs.uni-tuebingen.de/download/malt/welcome.html) for more information.